examples/displayio_layout_cartesian_dev.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: 2021 Stefan Krüger
+#
+# SPDX-License-Identifier: MIT
+#############################
+"""
+This is a basic demonstration of a Cartesian widget for line-ploting
```

Comment:
Do we need to keep both `displayio_layout_cartesian_dev.py` and `displayio_layout_cartesian_lineplot.py` ? They seem to be doing mostly the same thing. I think I would lean towards just keeping `displayio_layout_cartesian_lineplot.py` and if you want you can add the pointer behavior and different colors to it (though I think it's also fine to leave those out, either way is okay).

Reply:
yeah - thati will remove the *_dev.py.
and keep the *simple / minimal* plot.